src/SoftekBarcodeAPI.php

```diff
@@ -37,8 +40,101 @@ public function __construct(SoftekBarcodeConfig $configuration)
 
             // license the instance for further operations
             $this->ffi->mtSetLicenseKey($this->instance, $configuration->getLicenseKey());
-        } catch (\Throwable $e) {
-            throw new SoftekInitializationException($e->getMessage(), "", $e);
+        } catch (Throwable $e) {
+            throw new SoftekInitializationException($e->getMessage(), 0, $e);
+        }
+    }
+
+    /**
+     * Scan the specified image file for bar code strings and return the detected barcodes in the file as Objects.
+     * Note that the function will stop when the first barcode is found in a document unless
+     * the MultipleRead property is set to True
```

Comment:
To me, a user of this library not inspecting this exact source code, this is confusing. What is `MultipleRead`, where can I set it?

Reply:
Its quit hard to understand this, im agreed with you. Change will be done

Reply:
Seems like we should add this as simple boolean parameter.

In the future we should then add test cases for this parameter.